drivers/sensors/fakesensor_uorb.c

```diff
@@ -197,6 +197,19 @@ static inline void fakesensor_read_gyro(FAR struct fakesensor_s *sensor,
                     sizeof(struct sensor_gyro));
 }
 
+static inline void fakesensor_read_baro(FAR struct fakesensor_s *sensor,
+                                        uint64_t event_timestamp)
+{
+  struct sensor_baro baro;
+  char raw[50];
+  fakesensor_read_csv_line(
+          &sensor->data, raw, sizeof(raw), sensor->raw_start);
+  sscanf(raw, "%f,%f\n", &baro.pressure, &baro.temperature);
+  baro.timestamp = event_timestamp;
+  sensor->lower.push_event(sensor->lower.priv, &baro,
+                           sizeof(struct sensor_baro));
+}
```

Comment:
Sorry the indentation is a bit odd, could we change to this? Thanks :-)
```c
  struct sensor_baro baro;
  char raw[50];

  fakesensor_read_csv_line(&sensor->data, raw, sizeof(raw),
                           sensor->raw_start);
  sscanf(raw, "%f,%f\n", &baro.pressure, &baro.temperature);
  baro.timestamp = event_timestamp;
  sensor->lower.push_event(sensor->lower.priv, &baro,
                           sizeof(struct sensor_baro));
```

Reply:
Updated the formatting to exactly as you've requested. Thanks for the feedback!